StyleCop.Analyzers/StyleCop.Analyzers/Helpers/DocumentationCommentExtensions.cs

```diff
@@ -20,11 +20,22 @@ public static DocumentationCommentTriviaSyntax GetDocumentationCommentTriviaSynt
                 return null;
             }
 
-            return node
-                .GetLeadingTrivia()
-                .Select(i => i.GetStructure())
-                .OfType<DocumentationCommentTriviaSyntax>()
-                .FirstOrDefault();
+            foreach (var leadingTrivia in node.GetLeadingTrivia())
+            {
+                if (!leadingTrivia.HasStructure)
```

Comment:
:question: Is this condition helpful? It seems like we could simply remove it and everything else work continue to work the same way.